Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -89,23 +94,36 @@ public struct Configuration {
     }
 
     public init?(yaml: String) {
-        let yamlResult = Yaml.load(yaml)
```

Comment:
Yes, it was intentional. I refactored the initialization code so that it doesn't `Yaml.load` twice and `init` twice every time you initialize from a *.yml. This was previously done to detect errors in loading the *.yml and revert to a clean config if possible. The new approach separates the loading of the *yml (and the printing of the loading errors) into a separate method so that `Yaml.load` and `init` are only called once per initialization. It still preserves the error printing.